core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1763,6 +1767,10 @@ object SnappySession extends Logging {
         (rdd, findShuffleDependencies(rdd).toArray, rdd.id, false)
     }
 
+    val allallbroadcastplans = session.getContextObject[mutable.Map[BroadcastHashJoinExec,
```

Comment:
Yes it was being usd. Will keep it around as in the next phase of Tokenization this might be taken up.